src/styles/blocks/header.scss

```diff
@@ -0,0 +1,82 @@
+.catalog {
+  --colomn--count: 1;
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--colomn--count), 200px);
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    --colomn--count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    --colomn--count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    --colomn--count: 4;
+  }
+}
```

Comment:
Move this styles to the separate file, now you write this styles in the header styles file, but catalog isn't a header